CDPHW1/src/com/epam/cdp/java_testng/iryna_ramanets/task3/Runner.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.cdp.java_testng.iryna_ramanets.task3;
+
+import java.util.Scanner;
+
+public class Runner {
+
+	/**
+	 * @param msg to display for input
+	 * @return input
+	 */
+	public static String getInput(String msg) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print(msg);
+        String input = scanner.nextLine();
+        
+        if("q".equals(input)) {
+        	scanner.close();
+        	System.exit(0);
```

Comment:
Why exit so hard and not clear from maim method? maybe you should think about some flag that help finish program in more natural way? You can check q in while block